src/common/mesh_utils.cpp

```diff
@@ -50,4 +50,108 @@ std::shared_ptr<mfem::ParMesh> buildParallelMesh(const std::string& mesh_file, c
   return par_mesh;
 }
 
+// a transformation from the unit disk/sphere (in L1 norm) to a unit disk/sphere (in L2 norm)
+void squish(mfem::Mesh& mesh)
+{
+  int num_vertices = mesh.GetNV();
+  int dim          = mesh.SpaceDimension();
+
+  mfem::Vector vertices;
+  mesh.GetVertices(vertices);
+  mfem::Vector vertex(dim);
+  for (int i = 0; i < num_vertices; i++) {
+    for (int d = 0; d < dim; d++) {
+      vertex(d) = vertices[d * num_vertices + i];
+    }
+
+    double l1 = vertex.Norml1();
```

Comment:
This looks very similar to `11` and `12`. Could you rename to `l1_norm` and `l2_norm`?